examples/quickstart-xgboost/server.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+import flwr as fl
+from strategy import XGbBagging
+
+pool_size = 2
+num_rounds = 5
+num_clients_per_round = 2
+min_evaluate_clients = 2
+
+
+def eval_config(rnd: int) -> Dict[str, str]:
+    """Return a configuration with global epochs."""
+    config = {
+        "global_round": str(rnd),
+    }
+    return config
+
+
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    auc_aggregated = sum([metrics["AUC"] for _, metrics in eval_metrics]) / len(
+        eval_metrics
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
+    return metrics_aggregated
```

Comment:
Does it mean that that AUC is not influenced by the number of samples i.e. it's a simple average, not a weighted one? Is that intended behavior?
Let's add a comment if it's not a weighted avg that this is an expected behavior.

Reply:
The AUC values should be influenced by the number of samples. i'll double check this